shell/common/gin_converters/osr_converter.cc

```diff
@@ -143,19 +143,21 @@ v8::Local<v8::Value> Converter<electron::OffscreenSharedTextureValue>::ToV8(
           // texture, output it in second pass callback.
           data.SetSecondPassCallback([](const v8::WeakCallbackInfo<
                                          OffscreenReleaseHolderMonitor>& data) {
-            auto* iso = data.GetIsolate();
```

Comment:
Removed `iso` on purpose?

Reply:
Yes, It's unclear that this isolate is valid to be bind to the deferred task, e.g. mayb it would be invalidated after gc completes? Didn't go that deep to investigate.